modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-0_{context}"]
+= Release notes for {gitops-title} 1.13.0
+
+{gitops-title} 1.13.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.0_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-13-0-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.13.0 security update advisory
+
+Issued:
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-13-0_{context}"]
+== New features
+
+* With this update, you can configure Argo CD to disable the automatic scraping of metrics for your instance. Before this update, there was no option to disable this feature, resulting in excessive storage usage when multiple Argo CD instances were available on the cluster. link:https://issues.redhat.com/browse/GITOPS-4519[GITOPS-4519]
```

Comment:
Just wanted to add this comment to verify if we use Argo CD or ArgoCD - whatever is used for consistency in the docs. 
Otherwise LGTM, thanks @Dhruv-Soni11

Reply:
Hi @saumeya - I just quickly searched for instances of Argo CD and ArgoCD in the previous release notes and I found out that there were more occurrences of the term Argo CD used in the GitOps documentation in paragraphs and instances of ArgoCD are mostly likely found in the code snippet. So, I guess it makes sense to stick to Argo CD for release notes text and ArgoCD in code blocks or snippets. WDYT?

Reply:
Awesome, yes makes sense then.
There is one change I had requested on line 41, can you revert that back to Argo CD.

Reply:
The way I differentiate between these is, if I am talking about something in ArgoCD CRD/CR maybe a field or some config, I use term `ArgoCD` and if its something regarding Argo CD maybe a feature or configurations, I use `Argo CD`.